Content.Client/_Goobstation/Clothing/ToggleableClothingRadialMenu.xaml.cs

```diff
@@ -0,0 +1,96 @@
+using Content.Client.UserInterface.Controls;
+using Content.Shared.Clothing.Components;
+using Robust.Client.UserInterface;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.XAML;
+using Robust.Shared.Prototypes;
+using System.Numerics;
+
+namespace Content.Client._Goobstation.Clothing;
+
+public sealed partial class ToggleableClothingRadialMenu : RadialMenu
+{
+    [Dependency] private readonly EntityManager _entityManager = default!;
+
+    public event Action<EntityUid>? SendToggleClothingMessageAction;
+
+    public EntityUid Entity { get; set; }
+
+    public ToggleableClothingRadialMenu()
+    {
+        IoCManager.InjectDependencies(this);
+        RobustXamlLoader.Load(this);
+    }
+
+    public void SetEntity(EntityUid uid)
+    {
+        Entity = uid;
+        RefreshUI();
+    }
+
+    public void RefreshUI()
+    {
+        // Even EmotesMenu has to call this, I'm assuming it's essential.
+        var main = FindControl<RadialContainer>("Main");
+
+        if (!_entityManager.TryGetComponent<ToggleableClothingComponent>(Entity, out var clothing)
+            || clothing.Container is not { } clothingContainer)
+            return;
+
+        foreach (var attached in clothing.ClothingUids)
+        {
+            // Change tooltip text if attached clothing is toggle/untoggled
+            var tooltipText = Loc.GetString("toggleable-clothing-unattach-tooltip");
+
+            if (clothingContainer.Contains(attached.Key))
+                tooltipText = Loc.GetString("toggleable-clothing-attach-tooltip");
```

Comment:
```suggestion
            var tooltipText = Loc.GetString(clothing.UnattachTooltip);

            if (clothingContainer.Contains(attached.Key))
                tooltipText = Loc.GetString(clothing.AttachTooltip);
```
Hardcoded strings bad, these should be made [DataField]s on the ToggleableClothingComponent.

Reply:
Good point